Templates/GenerateTestValues.tpl

```diff
@@ -62,9 +63,10 @@ namespace Services.Test.GenerateValues
             ;Get by primary key
             repeat
             begin
-                read(chin,<structureNoplural>,^LAST) [ERR=eof<StructureNoplural>2]
+                read(chin,<structureNoplural>,^FIRST) [ERR=eof<StructureNoplural>2]
```

Comment:
is this template even used for non isam stuff... I think this is reverting a change